Kontent.Ai.Management/IManagementClientEarlyAccess.cs

```diff
@@ -0,0 +1,20 @@
+using Kontent.Ai.Management.Models.Shared;
+using Kontent.Ai.Management.Models.VariantFilter;
+using System.Threading.Tasks;
+
+namespace Kontent.Ai.Management;
+
+/// <summary>
+/// Represents a set of early access Content Management API requests.
```

Comment:
I think we call it simply Management API, not Content Management API. This naming is present in other places as well.